fota/full.yaml

```diff
@@ -0,0 +1,20 @@
+# Code generated by moulin. All manual changes will be lost
```

Comment:
If it is autogenerated, why it is committed?

Reply:
Sorry for the misunderstanding, let me make this clear.
I didn't use moulin yaml file to build AosEdge image, but I based on the structure of the moulin yaml to write this full.yaml in order to create a fota_bundle for FOTA operation for Gen5 virtual device. However, I forgot to remove the comment "Code generated by moulin. All manual changes will be lost". I will correct it.

Reply:
Why didn'you generate it via Moulin?

Reply:
My team is used to using Bash script and Docker to configure and build Yocto image. We planned to export a Moulin full.yaml file to build the AosEdge image once finishing the review of this PR. However, if you want to have a Moulin file now, we will create one and push to this PR.

Reply:
Yes, please add to this PR so that we have a single way if building all supported products & platforms

Reply:
Ok, I will add the Moulin file to this PR.